dashboard/app/models/pd/application/teacher_application_base.rb

```diff
@@ -662,7 +662,7 @@ def get_first_selected_workshop
 
     # @override
     def self.can_see_locked_status?(user)
```

Comment:
What's the base class for this in application? I suggest making that false, and allowing facilitator application to override it